external/platform/stm32f4xx/STM32F4xx_StdPeriph_Driver/CMSIS/stm32f4xx.h

```diff
@@ -50,6 +50,8 @@
  extern "C" {
 #endif /* __cplusplus */
   
+#define assert_param(expr) ((void)0)

```

Comment:
Can you leave a comment here as to what it's for? It's possible this a good define that we want to pull into LK itself.

Reply:
St code depends on assert_param(expr) but it is not defined in lk.

Reply:
https://github.com/littlekernel/lk/blob/a319a1f68d592529c03a1e95c369ffc0a2ed20cd/external/platform/stm32f0xx/STM32F0xx_HAL_Driver/stm32f0xx_ll_tim.c#L44